src/main/resources/config/node.json

```diff
@@ -1,7 +1,7 @@
 {
     "library": {
         "phone_number_capabilities": {
-            "PhoneNumberCapabilities": "interfaces"
+            "PhoneNumberCapabilities": "lib/interfaces"
```

Comment:
Why do we have to add `lib/`? PhoneNumberCapabilities are only used by files within lib folder.

Reply:
Just synching with the actual file here which includes the lib path: https://github.com/twilio/twilio-oai/blob/main/types/node.json